mmpose/datasets/pipelines/shared_transform.py

```diff
@@ -140,3 +141,94 @@ def __repr__(self):
         """Compute the string representation."""
         return (f'{self.__class__.__name__}('
                 f'keys={self.keys}, meta_keys={self.meta_keys})')
+
+
+@PIPELINES.register_module()
+class HideAndSeek:
+    """Augmentation by informantion dropping in Hide-and-Seek paradigm. Paper
```

Comment:
To help future readers and users, please provide a desription on difference between this implementation and the original paper's version, and also add reference to the original paper.

Reply:
I think it is unnecessary and improper to stated this here. As this should be found in Paper ref: Huang et al. AID: Pushing the Performance Boundary of Human Pose Estimation with Information Dropping Augmentation (arXiv:2008.07139 2020). This paper has detailed the relationship.